runtime/wasm/dune

```diff
@@ -55,10 +55,6 @@
   (glob_files *.wat))
  (action
   (progn
-   (bash
-    "which wasm-merge > /dev/null || (echo 'Error: Binaryen tools not found in the PATH'; false)")
-   (bash
-    "wasm-merge --version | grep -q 'version \\(11[89]\\|1[2-9][0-9]\\)' || (echo 'Error: Binaryen version 118 or greater is currently required'; false)")
```

Comment:
This doesn't work when running locally. I'm not sure why. Maybe we can replace it with a small ocaml program. or just drop it.

Reply:
I think we can just drop this. This is a lot less useful now that binaryen is installed by opam.